From f78f3c4c2ba6b5c2e339867ac84d0c9139f752a9 Mon Sep 17 00:00:00 2001
From: Yuntao Wang <yuntao.wang@linux.dev>
Date: Sat, 18 Jul 2026 12:01:46 +0800
Subject: [PATCH 10964/15333] kbuild: set the initial value of
 subdir-rustflags-y

Initialize subdir-rustflags-y to an empty value to prevent it from being
inadvertently affected by an environment variable of the same name.

Signed-off-by: Yuntao Wang <yuntao.wang@linux.dev>
Link: https://patch.msgid.link/20260718040146.294700-1-yuntao.wang@linux.dev
Signed-off-by: Nicolas Schier <nsc@kernel.org>
---
 scripts/Makefile.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/Makefile.build b/scripts/Makefile.build
index d432693e5367..a48209591dee 100644
--- a/scripts/Makefile.build
+++ b/scripts/Makefile.build
@@ -28,6 +28,7 @@ ldflags-y  :=
 
 subdir-asflags-y :=
 subdir-ccflags-y :=
+subdir-rustflags-y :=
 
 # Read auto.conf if it exists, otherwise ignore
 -include $(objtree)/include/config/auto.conf
-- 
2.55.0

